MapleServer2/Types/Player.cs

```diff
@@ -167,6 +167,7 @@ public JobCode JobCode
 
     public Character FieldPlayer;
 
+    private Dictionary<int, short> PassiveSkillEffects = new();
```

Comment:
Move this to skill tab?

Reply:
This was made to track active additional effects for whichever tab was active rather than for a specific tab. It's used to control how additional effects are set.